src/main/java/com/mongodb/jdbc/MongoBsonValue.java

```diff
@@ -119,6 +134,32 @@ public String toString() {
         }
     }
 
+    // Formats a BSON binary object into a JSON string representation of a UUID.
+    // If the BSON binary type is UUID_STANDARD, it directly converts it to a UUID.
+    // Otherwise, it uses the specified or default UUID representation to decode the binary data.
+    private String formatUuid(BsonBinary binary) {
+        UUID uuid;
+
+        if (binary.getType() == BsonBinarySubType.UUID_STANDARD.getValue()) {
+            uuid = binary.asUuid();
+        } else {
+            // When UNSPECIFIED, set UuidRepresentation to STANDARD
+            UuidRepresentation representationToUse =
+                    this.uuidRepresentation != UuidRepresentation.UNSPECIFIED
+                            ? this.uuidRepresentation
+                            : UuidRepresentation.STANDARD;
+            uuid =
+                    UuidHelper.decodeBinaryToUuid(
+                            binary.getData(), binary.getType(), representationToUse);
+        }
+
+        String uuidStr = uuid.toString();
```

Comment:
This is something Alexi and I have been trying to decide on.  It is a better experience to have only the `uuid` in the field for use in the query instead of having to add SUBSTRING() commands to remove the `{$uuid: }` parts.  Tableau does not have the easy transform commands as PowerBI has, at least from what we tried.